server/testserver.go

```diff
@@ -137,11 +137,9 @@ func (ts *TestServer) Start() error {
 
 	if !ts.SkipBootstrap {
 		stopper := util.NewStopper()
-		for _, eng := range ts.Ctx.Engines {
-			_, err := BootstrapCluster("cluster-1", eng, stopper)
-			if err != nil {
-				return util.Errorf("could not bootstrap cluster: %s", err)
-			}
+		_, err := BootstrapCluster("cluster-1", ts.Ctx.Engines, stopper)
```

Comment:
I think it's supposed to work to run `BootstrapCluster` on the first store and then let `ts.Server.Start()` (a few lines below) bootstrap any stores that aren't already bootstrapped. Was that not working or did you not think to try increasing StoresPerNode without changing BootstrapCluster?